docs/lib/graphqlapi/fragments/ios/query-data.md

```diff
@@ -121,3 +120,41 @@ func listTodos() -> AnyCancellable {
 </amplify-block>
 
 </amplify-block-switcher>
+
+> **Note**: This approach will only return up to the first 1,000 items.  To change this limit or make requests for additional results beyond this limit, use *pagination* as discussed below.
+
+## List subsequent pages of items
+
+A list query only returns the first 1,000 items by default, so for large data sets, you'll need to paginate through the results.  After receiving a page of results, you can check if there are subsequent pages and obtain the next page. The page size is configurable as well, as in the example below.
+
+
+```swift
+func listTodos() {
+    let todo = Todo.keys
+    let predicate = todo.name == "my first todo" && todo.description == "todo description"
+    Amplify.API.query(request: .paginatedList(Todo.self, where: predicate, limit: 1000)) { event in
+        switch event {
+        case .success(let result):
+            switch result {
+            case .success(let todo):
+                print("Successfully retrieved list of todos: \(todo)")
+                if todo.hasNextPage() {
```

Comment:
What is the type of `todo`?  Is it a single todo, or a list of todos?    Maybe this should be named `todos`?

Reply:
good callout- updating to plural